Sources/TuistGraph/Models/SwiftPackageManagerDependencies.swift

```diff
@@ -10,4 +10,58 @@ public struct SwiftPackageManagerDependencies: Equatable {
     public init(_ packages: [Package]) {
         self.packages = packages
     }
+
+    /// Returns `Package.swift` representation.
+    ///
+    /// **NOTE** It is a temporary solution until Apple resolves: https://forums.swift.org/t/pitch-package-editor-commands/42224
+    public func stringValue() -> String {
+        """
+        // swift-tools-version:5.3
+
+        import PackageDescription
+
+        let package = Package(
+            name: "PackageName",
+            dependencies: [
+                \(packages.map { $0.stringValue + "," }.joined(separator: "\n        "))
+            ]
+        )
+        """
+    }
+}
+
+// MARK: - Package.stringValue()
+
+private extension Package {
+    /// Returns `Package.swift` representation.
+    var stringValue: String {
+        switch self {
+        case let .local(path):
+            return #".package(path: "\#(path)")"#
+        case let .remote(url, requirement):
+            return #".package(url: "\#(url)", \#(requirement.stringValue))"#
+        }
+    }
+}
+
+// MARK: - Requirement.stringValue()
+
+private extension Requirement {
```

Comment:
Similar comment here. I'd move `stringValue` to `Requirement.swift` _(without extensions)_ and rename it to manifest value.

Reply:
This was actually my suggestion - in the future, we want to drop this logic altogether when it's possible to use CLI to add package dependencies (which should most likely come with Xcode 13, although, one can never be 100 % sure) and this is a way how to contain this logic. 

If we should ever need it somewhere else, I'd be for moving it to `Requirement.swift`, but for now I believe it makes more sense to keep it here as it's the only place where it's needed.

Alternatively, we could introduce private helper func to `SwiftPackageManagerDependencies` that will be along the lines of `manifestValue(for requirement: Requirement)` instead.